apis/nucleus/src/object/create-session-object.js

```diff
@@ -39,14 +39,15 @@ import init from './initiate';
  * nebbie.render(createConfig);
  */
 export default async function createSessionObject(
-  { type, version, fields, properties, options, plugins, element, extendProperties },
+  { type, version, fields, properties, options, plugins, element, extendProperties, navigation: inputNavigation },
   halo,
   store
 ) {
   let mergedProps = {};
   const children = [];
   const { modelStore, subscribe } = store;
   let error;
+  const navigation = inputNavigation;
```

Comment:
No need to bring it in as `inputNavigation` here and assign it to `navigation` before returning. Might as well just take in `navigation` and return the same, if I'm not missing anything...

Reply:
Or, actually, I don't see that we pass in navigation into this fn at all, so probably can remove it from here completely?